src/CountUp.js

```diff
@@ -71,7 +71,10 @@ class CountUp extends Component {
     // restart it.
     if (
       this.props.duration !== prevProps.duration ||
-      this.props.start !== prevProps.start
```

Comment:
This will work only if we provide **redraw** property. May be we should also add checks in sCU method, I'm not sure...

Reply:
The idea is not to need the `redraw` prop for changes in the `suffix`, `prefix`, `separator`. Do you think it should be enforced if `redraw` is passed for changes in the previous use-case?

Reply:
@Kenny407, if so, we should add this kind of checks in **shouldComponentUpdate** method also